datahub-web-react/src/app/ingestV2/executions/ExecutionsTab.tsx

```diff
@@ -0,0 +1,143 @@
+import { useCommandS } from '@app/ingestV2/shared/hooks/useCommandS';
+import { Message } from '@app/shared/Message';
+import { scrollToTop } from '@app/shared/searchUtils';
+import usePagination from '@app/sharedV2/pagination/usePagination';
+
+import { useListIngestionExecutionRequestsQuery } from '@graphql/ingestion.generated';
+import { ExecutionRequest } from '@types';
+
+const SourceContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    overflow: auto;
+`;
+
+const HeaderContainer = styled.div`
+    flex-shrink: 0;
+`;
+
+const StyledTabToolbar = styled.div`
+    display: flex;
+    justify-content: space-between;
+    padding: 0 0 16px 0;
+    height: auto;
+    z-index: unset;
+    box-shadow: none;
+    flex-shrink: 0;
+`;
+
+const TableContainer = styled.div`
+    flex: 1;
+    overflow: auto;
+`;
+
+const PaginationContainer = styled.div`
+    display: flex;
+    justify-content: center;
+    flex-shrink: 0;
+`;
+
+const DEFAULT_PAGE_SIZE = 25;
+
+export const ExecutionsTab = () => {
+    const [appliedFilters, setAppliedFilters] = useState<Map<string, string[]>>(new Map());
+    const [executionRequestUrnToView, setExecutionRequestUrnToView] = useState<undefined | string>(undefined);
+    const [hideSystemSources, setHideSystemSources] = useState(true);
+
+    const { page, setPage, start, count } = usePagination(DEFAULT_PAGE_SIZE);
+    // When filters changed, reset page to 1
+    useEffect(() => setPage(1), [appliedFilters, setPage]);
+
+    /**
+     * Show or hide system ingestion sources using a hidden command S command.
+     */
+    useCommandS(() => setHideSystemSources(!hideSystemSources));
+
+    const { filters, hasAppliedFilters } = useFilters(appliedFilters, hideSystemSources);
+    const { loading, error, data, refetch } = useListIngestionExecutionRequestsQuery({
+        variables: {
+            input: {
+                start,
+                count,
+                query: undefined,
+                filters,
+            },
+        },
+    });
+
+    const totalExecutionRequests = data?.listExecutionRequests?.total || 0;
+    const executionRequests: ExecutionRequest[] = data?.listExecutionRequests?.executionRequests || [];
+
+    // refresh the data when there are some running execution requests
+    useRefresh(executionRequests, refetch);
+
+    const onPageChangeHandler = useCallback(
+        (newPage: number) => {
+            scrollToTop();
+            setPage(newPage);
+        },
+        [setPage],
+    );
+
+    return (
+        <>
+            {error && (
+                <Message type="error" content="Failed to load execution requests! An unexpected error occurred." />
+            )}
+            <>
+                <SourceContainer>
+                    <HeaderContainer>
+                        <StyledTabToolbar>
+                            <Filters onFiltersApplied={(newFilters) => setAppliedFilters(newFilters)} />
```

Comment:
unfortunately it looks like the UI vs CLI filters aren't working properly right now for this tab